deploy/compliance-operator/20-complianceOperator-Subscription.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "compliance-operator"
+  namespace: "openshift-compliance"
+spec:
+  channel: "4.6"
+  installPlanApproval: "Manual"
```

Comment:
we wanted to gate compliance operator versions for the compliance operator and since the operator subscribes to a `CatalogSource` in `openshift-marketplace` - `redhat-operators` that we dont control, the compliance-operator dev team recommended adding `installPlanApproval: "Manual"` . I am not a 100% if that would help us gate versions so I am trying it out in stg for now. This is not intended to go to prod yet, hence the `config.yml` selector.

Reply:
Added owners

Reply:
We can't manually approve installs on each individual cluster, so this isn't a sustainable way of doing things. We either need automatic, or we need to figure out another path (like we re-host the catalog? we would need to explore this).

This might be okay for testing, but we can't roll anything to the fleet with this as we'd need to manually touch every cluster to install or upgrade the operator.

Reply:
yeah I agree. I am going to set it to automatic for now even though I dont intend to put this on prod anytime soon.